docs/se/dbclient/01_introduction.adoc

```diff
@@ -16,10 +16,227 @@
 
 ///////////////////////////////////////////////////////////////////////////////
 
-= Helidon DB Client
+= About Helidon DB Client
 :toc:
 :toc-placement: preamble
 :description: Helidon DB Client
 :keywords: helidon, se, database, dbclient
 
-== This page is Under Construction and will be available soon
+
+The Helidon SE DB Client provides a unified, reactive API for working with databases in non-blocking way. 
+
+== Helidon DB Client Features Overview
+
+The DB Client simplfies how you work with databases by allowing the use of blocking JDBC drivers in your reactive application by wrapping a blocking driver in an executor service. Specifically, the Heldion DB Client provides the following:
+
+=== Unified API for data access and query +
```

Comment:
These are actually not headers. Please change it to bullet list.

Reply:
It is a bulleted list now.